app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
Actually testing that with a running container is tricky, I think it would work to recompile the assets, then commit the container to a new image and restart Huginn using the new image.

When you hardcode the modes you can use the huginnbuilder images.